Documentation/hacking.md

```diff
@@ -54,6 +54,19 @@ Replace $SRC with the absolute path to your rkt source code:
 $ sudo docker run -v $SRC:/opt/rkt -i -t golang:1.3 /bin/bash -c "apt-get update && apt-get install -y coreutils cpio squashfs-tools realpath && cd /opt/rkt && go get github.com/appc/spec/... && ./build"
 ```
 
+### Building systemd in stage1 from the sources
```

Comment:
I think the ordering here is out of whack - the "Alternative stage1 paths" section below includes a sort of introduction which would make more sense *before* this section imho